### ISSUE TITLE
Greetings :) 

### ISSUE THREAD

**Issue by arenstar** [Jayachandra360/STOCK-PREDECTION#1]
Im wanting to build something like this and came across your repo.

Would you be interested in discussing more with me?

Thanks

**Comment by Jayachandra360** [comment]
Yes David, what are goal state of project or what is the problem statement of project ?
